pinot-core/src/main/java/org/apache/pinot/core/util/QueryOptions.java

```diff
@@ -33,6 +33,7 @@
   private final boolean _responseFormatSQL;
   private final boolean _preserveType;
   private final boolean _skipUpsert;
+  private final boolean _prefetchBuffers;
```

Comment:
We don't really need this memory variable. Eventually we might want to make this class a util class